docs/source/reference/spawners.md

```diff
@@ -297,8 +297,8 @@ Additional variables can be specified via the `Spawner.environment` configuratio
 
 The process environment is returned by `Spawner.get_env`, which specifies the following environment variables:
 
-- JUPYTERHUB*SERVICE_URL - the \_bind* URL where the server should launch its HTTP server (`http://127.0.0.1:12345`).
-  This includes `Spawner.ip` and `Spawner.port`; _new in 2.0, prior to 2.0 IP, port were on the command-line and only if specified_
+- JUPYTERHUB*SERVICE_URL - the \_bind\* URL where the server should launch its HTTP server (`http://127.0.0.1:12345`).
+  This includes `Spawner.ip` and `Spawner.port`; \_new in 2.0, prior to 2.0 IP, port were on the command-line and only if specified*
```

Comment:
Still [renders a bit off](https://jupyterhub--4245.org.readthedocs.build/en/4245/reference/spawners.html#environment-variables-and-command-line-arguments). Maybe autoformatter's getting something mixed up with `_/*`?

```suggestion
- JUPYTERHUB_SERVICE_URL - the *bind* URL where the server should launch its HTTP server (`http://127.0.0.1:12345`).
  This includes `Spawner.ip` and `Spawner.port`; new in 2.0, prior to 2.0 IP, port were on the command-line and only if specified.
```